package.json

```diff
@@ -140,7 +140,6 @@
         "@box/languages": "^1.0.0",
         "@commitlint/cli": "^8.3.5",
         "@commitlint/config-conventional": "^8.3.4",
-        "@commitlint/travis-cli": "^8.3.5",
```

Comment:
line 120

Reply:
should it be a breaking change?

Reply:
it's not actually required to use node 12, but it is the recommended. Node 8 should still work but I'm going to update it to use `>=10.x`

Reply:
actually let me update that to a breaking change since engine does break people installing box-ui-elements